dev/com.ibm.ws.security.ready.service/src/com/ibm/ws/security/ready/internal/SecurityReadyServiceImpl.java

```diff
@@ -238,4 +240,16 @@ private void updateSecurityReadyState() {
     public boolean isSecurityReady() {
         return securityReady;
     }
+
+    /**
+     * Provides a CountDownLatch that can be used to wait for the
+     * security service as a whole to be ready
+     *
+     * @return CountDownLatch that countdowns when security service is ready to
+     *         process requests
+     */
+    @Override
+    public CountDownLatch getSecurityReadyCDL() {
+        return securityReadyCDL;
```

Comment:
See other comments, but I think it would be nicer for the callers if this was `await` instead of exposing the CDL.